legacy/firmware/transaction.c

```diff
@@ -1146,7 +1146,7 @@ bool get_ownership_proof(const CoinInfo *coin, InputScriptType script_type,
   // Write versionMagic (4 bytes).
   memcpy(out->ownership_proof.bytes + r, &SLIP19_VERSION_MAGIC,
```

Comment:
now that SLIP19_VERSION_MAGIC is an array, you shouldn't be taking a pointer to it i think
(last time I checked, `array` and `&array` end up pointing to the same place, but still)